Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -624,6 +626,19 @@ AttachmentElementEnabled:
     WebKit:
       default: false
 
+AttachmentWideLayoutEnabled:
+  type: bool
+  status: embedder
+  humanReadableName: "Attachment wide-layout styling"
+  humanReadableDescription: "Use horizontal wide-layout attachment style, implies Attachment Element"
+  webcoreBinding: DeprecatedGlobalSettings
```

Comment:
We should avoid new uses of `DeprecatedGlobalSettings`.

You can remove this line, and add a:

```
WebCore:
      default: false
```

Then, access the new setting via `Settings` (which is accessible through `Document`).